server/data/voters.js

```diff
@@ -1,22 +1,44 @@
 const voters = [
     {
         "id": 1,
-        "voterName": "Do something nice for someone I care about",
+        "voterName": "Check something 1",
         "hasVoted": false,
+        "electionId": 123,
+        "secretBallot" : "eeeaaa",
+        "hasRegistered": true,
+        "isElectionOwner": true,
         "priority": "Important",
```

Comment:
what is this?

Reply:
This is just a sample json to put into DB. Just to make sure it's getting inserted as expected